runtime/src/test/scala/akka/grpc/GrpcClientSettingsSpec.scala

```diff
@@ -38,9 +51,16 @@ class GrpcClientSettingsSpec extends WordSpec with Matchers {
       parsed.host should be("myhost")
       parsed.port should be(42)
       parsed.overrideAuthority should be(Some("google.fr"))
-      parsed.trustedCaCertificate should be(Some("ca.pem"))
+      parsed.sslContext should be(None)
```

Comment:
All the other settings are simple types like `String`, `Int` or `Duration`. That almost makes me feel that `SSLContext` is the wrong type of object for us and we need something like an `SSLConfig` object. However, having an object like that, is pending on a decision about which API Akka gRPC wants to expose for configuring SSL. For now `SSLContext` is the least opinionated way for us to configure SSL.

Reply:
Having convenience constructors to use a `SSLConfig` to create the `SSLContext` here could make sense - that would mean pulling in ssl-config as a dependency, but still not requiring it to be used if the user wants something more flexible. I'm not sure we should go so far to make it the only way, though.